torch/_tensor_docs.py

```diff
@@ -1784,11 +1784,12 @@ def add_docstr_all(method, docstr):
 
 Args:
     dim (int): dimension along which to index
-    index (IntTensor or LongTensor): indices of :attr:`tensor` to select from
-    tensor (Tensor): the tensor containing values to add
+    index (Tensor): indices of ``source`` to select from,
+            should have type either `torch.int64` or `torch.int32`
```

Comment:
nit: "should have type" -> "should have dtype". "dtype" is PyTorch/numpy parlance for data type